tests/rptest/tests/delete_records_test.py

```diff
@@ -254,11 +279,15 @@ def test_delete_records_topic_start_delta(self, cloud_storage_enabled):
     # specific to delete-records but will happen in all cases where segment deletion
     # occurs at the moment a failure is injected.
     @cluster(num_nodes=3, check_for_storage_usage_inconsistencies=False)
-    @matrix(cloud_storage_enabled=[True, False],
-            truncate_point=[
-                "at_segment_boundary", "random_offset",
-                "one_below_high_watermark", "at_high_watermark"
-            ])
+    @matrix(
```

Comment:
seems like a lot of this is just formatting, if indeed this is compatible with any new standards then it should be in a seperate commit , otherwise use `yapf` to format

Reply:
I messed up a bit here. These changes shouldn't be present. Will force push amendments.